nmostesting/mocks/Registry.py

```diff
@@ -271,3 +417,192 @@ def heartbeat(version, node_id):
         return jsonify({"health": int(time.time())})
     else:
         abort(404)
+
+
+@REGISTRY_API.route('/x-nmos/query', methods=["GET"], strict_slashes=False)
+def query_root():
+    registry = REGISTRIES[flask.current_app.config["REGISTRY_INSTANCE"]]
+    if not registry.enabled:
+        abort(503)
+    authorized = registry.check_authorized(request.headers, request.path)
+    if authorized is not True:
+        abort(authorized)
+
+    base_data = ['v1.0/', 'v1.1/', 'v1.2/', 'v1.3/']
```

Comment:
This should reflect the enabled version as we discussed on the call.

Reply:
Code changed according to the discussion on handling API versions within the mocks